docs/standaard/documenten/release_notes.md

```diff
@@ -1,18 +1,55 @@
 ---
 title: "Release Notes Documenten API"
-date: '05-10--2023'
+date: '07-03-2024'
 weight: 10
 layout: page-with-side-nav
 ---
 # Release Notes Documenten API
 
+## Versie 1.5.0
+
+Versie   | Release datum 
+-------- | ------------- 
+1.5.0    | xx-03-2024    
+
+- property `inhoudIsVervallen` bij resource 'enkelvoudiginformatieobjecten' toegevoegd (zie toelichting hieronder).
+- regel die PUT en PATCH-operaties op 'enkelvoudiginformatieobjecten' alleen toestaat voor informatieobjecten waarvan de 'status NIET `definitief`' is ([drc-010](index.md#bijwerken-van-documenten-drc-010), is in overeenstemming met vervallen van deze regel in versie 1.4.0 uit de OAS verwijderd.
+
```

Comment:
Afsluitend haakje ontbreekt bij drc-010

Reply:
Sluithaakje ingevoegd.

Reply:
Verzoeken uit user story zijn voor een beperkt deel meegenomen in deze release. Dit toegelicht in milestone.